lumapps/api/exceptions/exception.py

```diff
@@ -0,0 +1,9 @@
+class InvalidUsage(Exception):
+    def __init__(self, message, status_code=400):
+        Exception.__init__(self)
+        self.message = message
+        self.status_code = status_code
```

Comment:
The status code here is not necessary as we are not a pure web app but a library we do not have vocation to carry that kind of informations.